src/store/index.ts

```diff
@@ -230,6 +230,7 @@ defaultTemplate.configTool.assignPort("out0", ConfigPortFunction.heater, 0);		//
 defaultTemplate.configTool.assignPort("out1", ConfigPortFunction.heater, 1);		// Nozzle heater
 defaultTemplate.configTool.assignPort("out3", ConfigPortFunction.fan, 0);			// Nozzle fan
 defaultTemplate.configTool.assignPort("out4", ConfigPortFunction.fan, 1);			// Nozzle thermostatic fan
+defaultTemplate.configTool.enableSTMBoards = true;
```

Comment:
The official ConfigTool cannot enable STM boards by default unless a disclaimer is confirmed stating that your boards are third-party boards and not developed or maintained by Duet3D. This disclaimer should show up when you click on the checkbox to enable STM boards.

Reply:
Yes I know but this PR is to discuss about the way I expanded the [Boards Record descriptor](https://github.com/TeamGloomy/configtool/blob/add-board-txt/src/store/Boards.ts#L54C14-L54C20).
I just forgot to remove this before the PR and I defined this default value to not have to check the checkbox each time I want to test something.
This will be indeed removed on the final PR.
This PR is not intended to be merged.